src/current/v24.1/row-level-ttl.md

```diff
@@ -226,7 +226,7 @@ CREATE TABLE events_using_timestamptz (
   start_date TIMESTAMPTZ DEFAULT now() NOT NULL,
   end_date TIMESTAMPTZ NOT NULL
 ) WITH (
-  ttl_expiration_expression = '((end_date AT TIME ZONE ''UTC'') + INTERVAL ''30 days'') AT TIME ZONE ''UTC'''
+  ttl_expiration_expression = '((end_date AT TIME ZONE ''UTC'') + ''30 days'') AT TIME ZONE ''UTC'''
```

Comment:
same as above

Reply:
this should be updated in latest changes, thanks!